x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/mappings_editor.tsx

```diff
@@ -27,123 +27,98 @@ interface Props {
 
 type TabName = 'fields' | 'advanced';
 
-export const MappingsEditor = React.memo(
-  ({ onUpdate, defaultValue = {}, indexSettings }: Props) => {
```

Comment:
@cjcenizal For info, in your merged PR (bb4a940f92784fb1df72ff671841e72e4fcca1e7) "Support _source field..." you introduced the set of the default value here (`defaultValue = {}`). As we watch for changes on the `defaultValue` in the `useMemo` below we can't do that as it creates a new object each time the parent rerenders.